crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -286,45 +287,44 @@ impl Node for BloomNode {
 
         // First downsample pass
         {
-            let downsampling_first_bind_group = {
-                let texture = BindGroupEntry {
-                    binding: 0,
-                    // We read from main texture directly
-                    resource: BindingResource::TextureView(view_target.main_texture()),
-                };
-
-                let sampler = BindGroupEntry {
-                    binding: 1,
-                    resource: BindingResource::Sampler(&pipelines.sampler),
-                };
-
-                let settings = BindGroupEntry {
-                    binding: 2,
-                    resource: uniforms.binding().unwrap().clone(),
-                };
-
+            let downsampling_first_bind_group =
                 render_context
                     .render_device
                     .create_bind_group(&BindGroupDescriptor {
                         label: Some("bloom_downsampling_first_bind_group"),
                         layout: &downsampling_pipeline_res.extended_bind_group_layout,
-                        entries: &[texture, sampler, settings],
-                    })
-            };
+                        entries: &[
+                            BindGroupEntry {
+                                binding: 0,
+                                // We read from main texture directly
+                                resource: BindingResource::TextureView(view_target.main_texture()),
+                            },
+                            BindGroupEntry {
+                                binding: 1,
+                                resource: BindingResource::Sampler(&pipelines.sampler),
+                            },
+                            BindGroupEntry {
+                                binding: 2,
+                                resource: uniforms.binding().unwrap().clone(),
```

Comment:
Oh no, I forgot to deal with this unwrap! Leaving this review here so I remember to do it